src/Microsoft.Health.Fhir.Core.UnitTests/Models/PartialDateTimeTests.cs

```diff
@@ -396,6 +397,19 @@ public void GivenAValidPartialDateTime_WhenToStringIsCalled_ThenCorrectStringSho
             Assert.Equal(expected, dateTime.ToString());
         }
 
+        [Theory]
+        [InlineData("de-DE", "2018-11-29T18:30:27.911+01:00", "2018-11-29T18:30:27,9110000+01:00")]
+        [InlineData("en-GB", "2018-11-29T18:30:27.911+01:00", "2018-11-29T18:30:27.9110000+01:00")]
+        [InlineData("en-US", "2018-11-29T18:30:27.911+01:00", "2018-11-29T18:30:27.9110000+01:00")]
+        public void GivenACulture_WhenToStringisCalled_ThenCorrectStringShouldBeReturned(string culture, string input, string expected)
+        {
+            System.Threading.Thread.CurrentThread.CurrentCulture = new System.Globalization.CultureInfo(culture);
```

Comment:
Can you add the constructor to the test to save the current culture and implement the IDisposable interface and restore the culture to original in Dispose?

Reply:
All done. Thank you